src/NUnitFramework/nunitlite.runner/OutputWriters/NUnit3XmlOutputWriter.cs

```diff
@@ -61,93 +61,85 @@ public override void WriteTestFile(ITest test, TextWriter writer)
         /// </summary>
         /// <param name="result">The result to be written to a file</param>
         /// <param name="writer">A TextWriter to which the result is written</param>
-        public override void WriteResultFile(ITestResult result, TextWriter writer)
+        public override void WriteResultFile(ITestResult result, TextWriter writer, IDictionary runSettings, TestFilter filter)
         {
-            XmlWriterSettings settings = new XmlWriterSettings();
-            settings.Indent = true;
+            XmlWriterSettings xmlSettings = new XmlWriterSettings();
+            xmlSettings.Indent = true;
 
-            using (XmlWriter xmlWriter = XmlWriter.Create(writer, settings))
+            using (XmlWriter xmlWriter = XmlWriter.Create(writer, xmlSettings))
             {
-                WriteXmlResultOutput(result, xmlWriter);
+                WriteXmlResultOutput(result, xmlWriter, runSettings, filter);
             }
         }
 
-        private void WriteXmlResultOutput(ITestResult result, XmlWriter xmlWriter)
+        private void WriteXmlResultOutput(ITestResult result, XmlWriter xmlWriter, IDictionary runSettings, TestFilter filter)
         {
-            this.xmlWriter = xmlWriter;
+            TNode resultNode = result.ToXml(true);
 
-            InitializeXmlFile(result);
+            // Insert elements as first child in reverse order
+            if (runSettings != null) // Some platforms don't have settings
+                FrameworkController.InsertSettingsElement(resultNode, runSettings);
```

Comment:
The nunitlite builds did not previously include the FrameworkController, but it is being called here.

Did I miss someplace that it was added to the nunitlite builds?